### PR TITLE
Moves headings to their own file - issue #670

### PR DESCRIPTION
**Pull request by knod**
Question: Should form-specific heading things be moved to a
different directory? A file in the 'forms' folder? I know we're
not using `ContentH1` anywhere other than forms right now, but
I hope to use it in the About page and maybe in our style guide,
other documentation, etc.

### CODE REVIEW

File: src/localization/de.js [hunk 1]
```diff
@@ -64,8 +64,8 @@ export default {
     whoMadeThis2C: ` aufrecht. Für weitere Auskunft treten Sie bitte in Verbindung mit `,
     whoMadeThis2D: `.`,
     
-    whoMadeThis3Intro: `Wir möchten den folgenden Freiwilligen von Code for Boston besonders danken: `,
-    whoMadeThis3Names: `Annie LaCourt, Isaac Chansky, Michelle Bernstein, Alec Danaher, Sasha Maryl, Drew Love, Liani Lye, Andrew Cunningham, Liam Morley, Nick Francisci, Stephen Chin, Shameek Poddar, Will McIntosh, Andrew Seeder, Ben Lewis, Don Blair, Ethan Strominger, Nick Lee, Jonathan Marcus, Emily Wasserman, Ethan Blackwood,`,
+    whoMadeThis3Intro:     `Wir möchten den folgenden Freiwilligen von Code for Boston besonders danken: `,
```
Comment (knod) [inline src/localization/de.js]: Damnit. This apparently needed fixing elsewhere. Can we just pull it in too? Bah, laziness on my part, but still.

Comment (ethanbb) [reply src/localization/de.js]: Yeah, I noticed I almost had the same changes in `de.js` in my last pull request too. It's just a whitespace thing, right?

Comment (knod) [reply src/localization/de.js]: Yeah.

**Comment by knod** [comment]
I'm going to clean this up and resolve conflicts, but I don't think this is the place to address the stuff we've been talking about for headings. This PR is basically just for rearranging into different files and that stuff is changing how the components are working together. We could make an issue for changing up the heading stuff, though.

**Pull request merged**